.github/workflows/run-tests.yml

```diff
@@ -10,22 +10,28 @@ jobs:
       fail-fast: true
       matrix:
         php: [7.3, 7.4, 8.0, 8.1, 8.2]
-        laravel: [^6.0, ^7.0, ^8.0, ^9.0]
+        laravel: [6, 7, 8, 9, 10]
         exclude:
           - php: 7.3
-            laravel: ^9.0
+            laravel: 9
+          - php: 7.3
```

Comment:
Please let's drop 7.3 support and tag a new major version

Reply:
To be honest, I don't see why we should drop support for PHP 7.3 right now. We are not using any features of PHP 7.4 or later and keeping the support for it does not make maintaining this package any harder.